### ISSUE TITLE
convert weight from tensorflow to Keras

### ISSUE THREAD

**Issue by Shiro-mx** [titu1994/MobileNetworks#5]
Hello,

I am trying to use your extract and load weight so as to convert the model from tensor flow to Keras. I have a problem during the using of the file load_weight.

```
Traceback (most recent call last):
  File "weight_load.py", line 60, in <module>
    model.layers[1].set_weights(conv1_weights)
  File "/usr/local/lib/python3.5/dist-packages/Keras-2.0.4-py3.5.egg/keras/engine/topology.py", line 1190, in set_weights
    'provided weight shape ' + str(w.shape))
ValueError: Layer weight shape (3, 3, 3, 8) not compatible with provided weight shape (128,)
```
I am trying to convert mobilenet_v1_0.25_224.ckpt.  In the weight load I have change this line : 

`model = MobileNets((128, 128, 3), alpha=0.25, weights=None)`
to 

`model = MobileNets((224, 224, 3), alpha=0.25, weights=None)`

Am I doing something wrong?

**Comment by titu1994** [comment]
Umm the weights are provided already. There is no need to convert them again. 

Just build and use model for inference, or use include_top=False for fine-tuning purposes. 

The default image size is 224x224

**Comment by titu1994** [comment]
Oops did not mean to close the issue. If you want to convert weights, you need to change the checkpoint name at the top of the file, and then the model parameters to match the checkpoint parameters (input sizes and alpha value)

**Comment by Shiro-mx** [comment]
thank for your reply. That's what I tried to do. I wanted to convert the mobile net's file whose the alpha is equal to 0.25 and the input size 224. But I got the previous error .

**Comment by titu1994** [comment]
But why do you want to convert the weights in the first place ? The weights are already provided for Keras.

**Comment by Shiro-mx** [comment]
Oh sorry, I thought the weights provided was available only for the version with alpha = 1.0
Thank for the help !

**Comment by titu1994** [comment]
No problem. But you did point out an actual bug, which was the number of classes is now 1000 from 1001. Thanks for that. I've fixed that bug.
